src/hatch_rest_api/riot.py

```diff
@@ -168,13 +168,18 @@ def turn_light_off(self):
         _LOGGER.debug(f"Turning light off")
         # 9999 = custom color 9998 = turn off
         self._update(
-            {
-                "current": {
-                    "color": {
-                        "id": 9998
+                {
+                    "current": {
+                        "playing": "none",
```

Comment:
I think we need to pass playing at whatever its current value is to prevent changing the sound.

Reply:
I guess this won't work like you mentioned below... There is no way to really keep them separate. When I get some time I'll double check what the Android app is actually sending and see if we can do something

Reply:
Yeah, there is no way to turn the light off in the app really just setting the brightness to 0 really. We could fake it out?

Basically if the sound is playing (playing != none) turn off by setting brightness to 0 and color ID to 9998. Turning on will set to 9999 and the previous brightness. Turning off won't change the audio. Stopping the audio will still turn off the light since we have no way to set the audio to the none track at this time. 

Thoughts?

Reply:
> Basically if the sound is playing (playing != none) turn off by setting brightness to 0 and color ID to 9998. Turning on will set to 9999 and the previous brightness. Turning off won't change the audio. Stopping the audio will still turn off the light since we have no way to set the audio to the none track at this time.

But the only way that a sound is playing would be if "playing = routine" because we are only pulling sound from the favorites. Therefore, if "playing = remote" then we are only using the light, so if "playing = remote" then if we turn off the light then we could change playing to "none" and RGBW to 0 then it would mimic pressing "Power Off" in the android and iOS app. If we just set the brightness to 0 then when you go into the app, maybe to change your favorites, then it is going to show that your Hatch is ON. Functionally, though yes either way would work and the end user wouldn't know the difference.

Reply:
I don't really care either way I really don't have plans to use the light independent from a favorite, but I think if we can get Home Assistant and the official app to play nice together we should try. I assume it's possible that it could be set to remote or some other value when playing one of the paid membership sounds? And if we wanted the user to be able to play one of those but change the light in HA we would have to not touch the audio when changing the light.